### DISCUSSION TITLE
Recommending LXQt 1.1 for Debian and Ubuntu

### DISCUSSION THREAD

**Discussion by tsimonq2** [lxqt/lxqt#2281] [Media]
You can't underestimate how many new ways the end user will find to break stuff, I guess. We got our first bug report today of someone who enabled the Lubuntu Backports and [this PPA](https://lxqt-project.org/blog/2022/05/06/latest-lxqt-debian-ubuntu/) at the same time.

That PPA uses versioning that directly conflicts with whatever is in the archive. The packaging hasn't been checked and I have no clue what bugs may lie in there.

As someone who officially holds hats in Lubuntu and Debian, I'm asking you guys to take the post down, or at least amend it, pretty please. Let me know how I can help.

**Comment by jonesmz** [comment]
As someone who is merely a user of lxqt and nothing more...

Why should lxqt take their blogpost down or amend it?

As someone who has hats in Lubuntu and Debian, you have more meaningful ability to fix whatever the problem is.

Either fix the problem in your distro(s), or open a pull request to whatever is driving the PPA that the blogpost talks about.

**Reply by stefonarch** [reply]
I updated the post now, I think it's better, as we spread the news we should also care for it afterwards.
https://github.com/lxqt/lxqt.github.io/pull/117

**Reply by stefonarch** [reply]
FYI: I tested the switch  of the ppa's in my VM, it needed:
```
apt update
apt full-upgrade <- this removes lxqt-session and some other components
apt install lxqt-core
```

**Reply by tsimonq2** [reply]
> I updated the post now, I think it's better, as we spread the news we should also care for it afterwards. [lxqt/lxqt.github.io#117](https://github.com/lxqt/lxqt.github.io/pull/117)

Thank you!

**Comment by stefonarch** [comment]
I  noticed that packages for  LXQt 1.1.0 have been uploaded to debian sid unstable.

**Reply by tsujan** [reply]
> packages for LXQt 1.1.0 have been uploaded to debian sid unstable.

6 months behind, in Debian Unstable?!

**Reply by stefonarch** [reply]
Better late than never, and a huge jump from 0.16 :)

**Reply by stefonarch** [reply]
![screen_area_dom_11:15:39_](https://user-images.githubusercontent.com/10681413/205485265-c711b9e9-510d-47da-a827-4ba672cbfd2f.png)
 Some days it was uploaded to experimental, now 1.2.0 is arriving in unstable.

**Comment by stefonarch** [comment]
Looks like it's in testing now.

**Comment by edwardp3** [comment]
After trying several distros in VirtualBoxes, I decided to go back to Debian (unstable/sid) and was waiting for the 1.2.0 packages to be available in the repository before installing LXQt.

**Reply by stefonarch** [reply]
Now Debian testing is a real good choice, it runs flawlessly on the "family" PC here for a year or so.

**Comment by edwardp3** [comment]
Unfortunately, I am unable to use the new LXQt.

I am now using the Vivaldi browser and for some unknown reason, its UI blinks if I try to access the menu, the same occurred with the XFCE desktop. Not sure what's causing it, but it works fine under LXDE. I removed the LXQt packages.

**Reply by stefonarch** [reply]
Could be well the compositor (+a bug in vivaldi), picom o compton was active? Same in xfce, you can disable it in xfwm4 settings.

**Reply by stefonarch** [reply]
https://forum.vivaldi.net/topic/8116/screen-flickers/8

**Comment by edwardp3** [comment]
The browser's UI was blinking, the rest of the screen was fine (and the tear-free option is enabled). 

I don't know what LXQt is/was using for a compositor. XFCE uses xfwm4 by default. But once LXQt was installed, I was not presented with a login option for Wayland, the default came up using X11.

**Reply by tsujan** [reply]
> I don't know what LXQt is/was using for a compositor

LXQt does not come with a compositor or window manager of its own. You can use any DE-independent window manager with LXQt, like Openbox, xfwm4, KWin,....

> I was not presented with a login option for Wayland

LXQt doesn't have a Wayland session (yet), but most of its components can be used under any Wayland session — like Wayfire, Sway,...

**Reply by stefonarch** [reply]
If I'm not mistaken debian choosed  to ship xfwm4 in LXQt. To disable it:

![screen_area_ven_21:58:18_](https://user-images.githubusercontent.com/10681413/209405068-06f7492a-80bf-4074-888c-54e60216c9cc.png)

**Reply by tsujan** [reply]
I haven't used Vivaldi but, IMO, if an app has problem with compositing in 2022, that problem should be fixed in the app, not by disabling compositing.

**Reply by stefonarch** [reply]
It's a wild guess, that's the compositor's fault. 
I just installed it, no issue on wayland.

**Reply by edwardp3** [reply]
I reinstalled the LXQt packages. Once the compositor was turned off as above, the UI was working perfectly. So it must be the compositor's fault, in this case, xfwm4.

**Reply by stefonarch** [reply]
You could try if using picom (ex compton) with xfwm4 has the same issue or not.

**Reply by edwardp3** [reply]
xfwm4 - the bahavior is intermittent. It started out the same, then apparently cleared up, menu entries are working fine after a while.

picom -  not only is the behavior the same initially, but I also lost the window title bar and the window tab did not appear in the LXQt panel. Screenshot attached showing the entire browser window. 

In addition, both the Window Manager and Window Manager Tweaks menu selections would _not_ launch with picom and there was also a delay before the desktop appeared after login. These two items will launch and there was no delay in desktop appearance, using xfwm4.

![Screenshot from 2022-12-24 10-03-37](https://user-images.githubusercontent.com/73129450/209441888-a1ee7b74-05e5-4fc0-9f46-abe8ca49682e.png)

**Reply by edwardp3** [reply]
Since I've been using more of LXQt with Vivaldi, I am no longer having any issues. Everything with LXQt is now working as expected.
